lib/full_node.js

```diff
@@ -0,0 +1,14 @@
+//      
+var bcoin = require('bcoin');
+var TrustIsRisk = require('./trust_is_risk');
+
+class FullNode extends bcoin.fullnode {
+                     
+
+  constructor(options         ) {
```

Comment:
The build script is already in package.json.

It's common to check in the compiled source code in compiled javascript libraries. I assume the reason is that a compiled javascript program is still cross-platform (unlike a binary), and that it's a bit more convenient to not have to install build tools (in this case Flow and flow-remove-types) when trying to run an older version of the library. On the other hand it does take up some space and looks ugly. I think it would be ideal if there were a way for git to treat the `lib/` folder like it treats binaries (e.g. calculate no diffs).